test_cli_remapping/test/test_cli_remapping.py

```diff
@@ -42,127 +40,133 @@ def get_environment_variable(name):
     get_environment_variable('NAME_MAKER_RCLPY')
 )
 
-
-@pytest.fixture(params=CLIENT_LIBRARY_EXECUTABLES)
-def node_fixture(request):
-    """Create a fixture with a node, name_maker executable, and random string."""
-    rclpy.init()
-    node = rclpy.create_node('test_cli_remapping')
-    try:
-        yield {
-            'node': node,
-            'executable': request.param,
-            'random_string': '%d_%s' % (
-                random.randint(0, 9999), time.strftime('%H_%M_%S', time.gmtime()))
-        }
-    finally:
-        node.destroy_node()
+TEST_CASES = {
```

Comment:
Probably out of the scope of this PR, but IMHO this migrated test looks a little difficult to read. I mean, I needed to read the comments in `launch/launch_testing/example_test_context.test.py` to understand what was happening here (maybe, there is better documentation somewhere).
Passing test argument in the returned test context it's not quite intuitive. Also, it would be nice to have better parameterization support.

Reply:
I know it's hard to read, I did my best to not split it up while keeping some of the `unittest` like structure. Alternatively, we could partially recreate what `pytest` does with decorators to inject the test functions like it used to be but it seemed a bit too much for a migration.